src/__tests__/__components__/AppBars/NavBar.test.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import toJson from "enzyme-to-json";
+import { shallow } from "enzyme";
+import { Navbar } from "../../../components/common/AppBars/navBar";
+import TextInput from "../../../components/common/Inputs/TextInput";
+import SearchPopOver from "../../../components/PopOvers/SearchPopOver";
+
+const [authSuggestArticles, fetchResults, push] = new Array(3).fill(jest.fn());
+
+const props = {
+  authSuggestArticles,
+  fetchResults,
+  searchQuery: "",
+  history: {
+    push,
+    location: {
+      pathname: "/"
+    }
+  },
+  suggestedArticles: {}
+};
+const wrapper = shallow(<Navbar {...props} />);
+jest.useFakeTimers();
+describe("NavBar component", () => {
+  describe("rendered component", () => {
+    test("should match the snapshot", () => {
```

Comment:
Please consider leaving an empty space between your tests for better readability.